README.md

```diff
@@ -112,6 +130,8 @@ Now we can generate URLs in our markup:
 <img class="img-responsive img-thumb" src="<%= Images.url(user.avatar, :thumb) %>" alt="">
 ```
 
+Note that ```user.avatar``` from the above example is the name of a file inside ```web/static/assets/images/``` and not a full path.
```

Comment:
I think we should word this differently, the value of `user.avatar` can be both a filename or a subpath.